### PR TITLE
Get CI working for 1.6

### PR DESCRIPTION
**Pull request by baude**
* switch to 1.6 gating image

Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -36,6 +36,7 @@ env:
     SPECIAL_FEDORA_CACHE_IMAGE_NAME: "xfedora-30-${_BUILT_IMAGE_SUFFIX}"
     UBUNTU_CACHE_IMAGE_NAME: "ubuntu-19-${_BUILT_IMAGE_SUFFIX}"
     PRIOR_UBUNTU_CACHE_IMAGE_NAME: "ubuntu-18-${_BUILT_IMAGE_SUFFIX}"
+    CENTOS_7_NAME: "centos-7-v20200618"
```
Comment (edsantiago) [inline .cirrus.yml:39]: I'm having a hard time understanding what this accomplishes. Is it a holdover from an earlier iteration of this PR? And if it is, would it make sense to remove the unused UBUNTU_* declarations?

Comment (edsantiago) [reply .cirrus.yml:39]: I really hate how github handles comments on stale diffs

Comment (baude) [reply .cirrus.yml:39]: yes, hold over

**Pull request merged**